components/Footer/Footer.tsx

```diff
@@ -124,6 +124,8 @@ const SocialMediaIcons = () => (
     </ul>
 );
 
+const Banner = () => <div className={styles.emptyEmailBanner}></div>;
+
 const EmailSignup = () => (
```

Comment:
**Suggestion**: This could be it's own component, happy for you to leave this though as it's not being used.